.rodare.json

```diff
@@ -102,6 +102,12 @@
             "affiliation": "NVIDIA",
             "name": "Kirkham, John",
             "type": "Other"
+        },
+        {
+            "affiliation": "Perimeter Institute for Theoretical Physics",
```

Comment:
@eschnett thank you for your contributions already!

We will push out this long-accumulated major release and then move your contributions to co-author once we merge your two open PRs in, if that is ok with you, too :)